src/scss/components/statistics.scss

```diff
@@ -0,0 +1,128 @@
+.statistics {
```

Comment:
Can you add the url to test this in localhost?

Reply:
Sure!! 

http://localhost:8080/linisabri.html